clang/test/CAS/mccas-replay-test.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: export LLVM_CACHE_CAS_PATH=%t/cas && %clang-cache \
+// RUN:   %clang -target x86_64-apple-macos11 -c -Xclang -fcas-backend -Rcompile-job-cache %s -o %t/tmp.o 2>&1 | FileCheck %s -check-prefix=CACHE-MISS
+// CACHE-MISS: remark: compile job cache miss
+
+// RUN: wc -c %t/tmp.o | FileCheck %s -check-prefix=CACHE-MISS-FILE
```

Comment:
I prefer either diff two files between cache hit/miss or between with/without cas-backend, or maybe using llvm-objdump or other object reader to check that is a valid object file.